### PR TITLE
[Bugfix] Fix postal validation

### PR DESCRIPTION
**Pull request by mwickett**
I missed limiting the length on the postal field. We had one entry with a full postal - I've updated that in the prod DB.

### CODE REVIEW

**Review by silent1mezzo — APPROVED** [review]

File: components/form/index.tsx [hunk 1]
```diff
@@ -186,7 +186,15 @@ export default function StoryForm() {
                 isInvalid={form.errors.postal && form.touched.postal}
               >
                 <FormLabel htmlFor="postal">First 3 characters of your postal code</FormLabel>
-                <Input {...field} id="postal" placeholder="P6A" size="lg" />
+                <Input
+                  name={field.name}
+                  value={field.value.toUpperCase()}
```
Comment (mwickett) [inline components/form/index.tsx:191]: Force the value to upper case on the form itself.

File: lib/api/stories.ts [hunk 2]
```diff
@@ -62,6 +62,9 @@ export async function add(story: NewStory) {
     // Strip empty values
     if (story[k] === '') {
       return acc
+      // Uppercase postal
+    } else if (k === 'postal') {
+      acc[k] = story[k].toUpperCase()
```
Comment (mwickett) [inline lib/api/stories.ts:67]: Also force it on the backend.

**Review by silent1mezzo — APPROVED** [review]
On mobile and haven't been able to test it out but code looks good

**Comment by mwickett** [comment]
I'm going to merge this since we're currently allowing lower case postals.

**Pull request merged**